app/jobs/datadog_sync_job.rb

```diff
@@ -0,0 +1,36 @@
+class DatadogSyncJob < ApplicationJob
+  queue_as :default
+
+  def perform(job)
+      conn = Faraday.new(
+        url: "https://api.datadoghq.com",
+      )
+      response = conn.get do |req|
+        req.url "/api/v2/users"
+        req.headers["Content-Type"] = "application/json"
+        req.headers["DD_API_KEY"] = job[:api_key]
+        req.headers["DD_APPLICATION_KEY"] = job[:application_key] # HAS TO BE DD_APPLICATION_KEY
+        req.headers["DD_SITE"] = job[:subdomain]
+      end
```

Comment:
We're getting data via 1 API call only. Can you handle the case where we cannot fetch all data in a single API? (Pagination)